modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase

```diff
@@ -2222,6 +2222,123 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-10750. User can see if a SKU is on order and can search for Order Id or Account Name the orders under Order Tab in Inventory"
+	@priority = 5
+	test CanViewOrderSKUAndSearchForSpecificOrderIdAndAccountName {
+		property portal.acceptance = "true";
+
+		task ("Given a Minium site with money order enable and two accounts are created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			CommerceChannels.openChannelsAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Minium Portal");
+
+			CommerceChannels.editPaymentMethod(
+				active = "true",
+				paymentMethod = "Money Order");
```

Comment:
@Brunno0liveira as this payment method was enabled in the setup, at this point in the test Money Order is already enabled, please remove these lines